Havit/ShareExtension/Screens/Category/View/SelectCategoryIconViewController.swift

```diff
@@ -0,0 +1,173 @@
+        let label = UILabel()
+        label.text = "아이콘"
+        label.font = .font(.pretendardSemibold, ofSize: 14)
+        label.textColor = .havitPurple
+        return label
+    }()
+    
+    private let iconCollectionView: UICollectionView = {
+        let collectionViewLayout = UICollectionViewFlowLayout()
+
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewLayout)
+        collectionView.register(cell: CategoryIconCollectionViewCell.self)
+        return collectionView
+    }()
+    
+    private let completeButton: UIButton = {
+        var container = AttributeContainer()
+        container.font = .font(.pretendardSemibold, ofSize: 16)
+        
+        var configuration = UIButton.Configuration.plain()
+        configuration.attributedTitle = AttributedString("완료", attributes: container)
+        configuration.background.cornerRadius = 0
+        configuration.baseForegroundColor = .white
+        configuration.contentInsets = NSDirectionalEdgeInsets(top: 20, leading: 173, bottom: 44, trailing: 174)
+        
+        let button = UIButton(configuration: configuration, primaryAction: nil)
+        
+        let buttonStateHandler: UIButton.ConfigurationUpdateHandler = { button in
+            switch button.state {
+            case .normal:
+                button.configuration?.background.backgroundColor = .havitPurple
+            case .disabled:
+                button.configuration?.background.backgroundColor = .gray002
+            default:
+                return
+            }
+        }
+        button.configurationUpdateHandler = buttonStateHandler
+        
+        return button
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setDelegation()
+    }
+    
+    // MARK: - func
+    
+    override func configUI() {
+        setNavigationBar()
+    }
+    
+    override func render() {
+        view.addSubViews([selectIconNoticeLabel,
+                          selectIconHeaderLabel,
+                          iconCollectionView,
+                          completeButton])
+        
+        selectIconNoticeLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(40)
+            $0.leading.equalToSuperview().inset(16)
+        }
+        
+        selectIconHeaderLabel.snp.makeConstraints {
+            $0.top.equalTo(selectIconNoticeLabel.snp.bottom).offset(34)
+            $0.leading.equalTo(selectIconNoticeLabel)
+        }
+        
+        iconCollectionView.snp.makeConstraints {
+            $0.top.equalTo(selectIconHeaderLabel.snp.bottom).offset(16)
+            $0.leading.trailing.equalToSuperview().inset(16)
+            $0.bottom.equalToSuperview()
+        }
+        
+        completeButton.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview()
+            $0.bottom.equalToSuperview()
```

Comment:
p1;

```suggestion
            $0.leading.trailing.bottom.equalToSuperview()
```
이렇게 쓸 수 있을 거 같아요!

Reply:
넵!! 반영하였습니다!
> 반영 commit : [링크](https://github.com/TeamHavit/Havit-iOS/pull/148/commits/d00fe0103668c2e5f3807db5ab91f9cb18ed88cb#diff-8a3d3911beb933dfcaf77772917924a6ccd875dc5834eb4726b177e0602c3451R125)